### PR TITLE
edit v3.5 blog post copy and pictures

### LINKED ISSUE #335: Complete v3.5-announcement blog review
This is the blog entry: https://github.com/etcd-io/website/blob/main/content/en/blog/2021/announcing-etcd-3.5.md

### PR DESCRIPTION
**Pull request by kapunahelewong**
- reduces size of images
- removes fig caption copy from images so it is editable and accessible
- adds alt text to images
- edits copy (one stylistic edit included compound possessives, such as "person@ of Google's fix". If that's not how we do it, I can revert. 🙂)

Fixes #335 

Ready for your review, @chalin and @nate-double-u!

### CODE REVIEW

**Comment by nate-double-u** [comment]
Looks like the link checker failed:

```
12:46:17 PM: htmltest started at 07:46:17 on public.htmltest
12:46:17 PM: ========================================================================
12:46:19 PM: blog/2021/announcing-etcd-3.5/index.html
12:46:19 PM:   target does not exist --- blog/2021/announcing-etcd-3.5/index.html --> github.com/ptabor
12:46:19 PM: ========================================================================
12:46:19 PM: ✘✘✘ failed in 1.898275854s
12:46:19 PM: 1 errors in 405 documents
```

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 1]
```diff
@@ -282,95 +273,79 @@ change](https://github.com/etcd-io/website/issues/149)). The migration was
...
-[github.com/wenjiaswe](https://github.com/wenjiaswe)) who's been leading etcd
-community meetings and Kubernetes integration, and Piotr Tabor (Google,
-[github.com/ptabor](https://github.com/ptabor)) who's been leading numerous bug
-fixes and codebase modularization work. The diversity of contributors is key to
+The diversity of etcd end users keeps expanding: Cloudflare [relies on etcd for managing its data center](https://blog.cloudflare.com/a-byzantine-failure-in-the-real-world),
+Grafana Cortex [stores its configuration data in etcd](https://grafana.com/blog/2020/04/07/how-a-production-outage-in-grafana-clouds-hosted-prometheus-service-was-caused-by-a-bad-etcd-client-setup),
+Netflix [Titus](https://netflix.github.io/titus/) uses etcd for [managing its container workloads](https://www.slideshare.net/aspyker/herding-kats-netflixs-journey-to-kubernetes-public), and
+Tailscale [runs its control plane on top of etcd](https://tailscale.com/blog/an-unlikely-database-migration).
+
+We have also extended team of our vendorcontributors.
```
Comment (gyuho) [inline content/en/blog/2021/announcing-etcd-3.5.md:289]: `s/vendorcontributors/vendor contributors/`

**Comment by gyuho** [comment]
I personally prefer self-contained images with caption, easier to read, if accessibility can be done by HTML caption tag. Other than that, looks great to me. Thanks for the improvements. Defer to @chalin @nate-double-u

**Comment by nate-double-u** [comment]
> Looks like the link checker failed

The issue was a link to `github.com/ptabor`, but the link checker wanted `https://github.com/ptabor`.

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-367--etcd.netlify.app/blog/2021/announcing-etcd-3.5/

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 2]
```diff
@@ -30,191 +30,190 @@ Given that etcd often handles sensitive data, improving and maintaining security
...
+**gRPC gateway endpoint is now stable under `/v3/*`**. gRPC gateway generates an
+HTTP API to make etcd gRPC-based HTTP/2 protocol accessible via HTTP/1, such as:
 
 > curl -X POST -L http://localhost:2379/v3/kv/put -d '{"key": "Zm9v", "value": "YmFy"}'
 
-**etcd client now uses latest gRPC v1.32.0 and migrated to upstream balancer
-implementation** -- see [ptabor@'s code change (from
-Google)](https://github.com/etcd-io/etcd/pull/12671), and **requires a new
-import path `"go.etcd.io/etcd/client/v3"`**.
+**etcd client now uses the latest gRPC, v1.32.0, **requires a new
```
Comment (gyuho) [inline content/en/blog/2021/announcing-etcd-3.5.md:90]: Missing closing `**`? Or remove `**` from `**requires`?

Comment (kapunahelewong) [reply content/en/blog/2021/announcing-etcd-3.5.md]: Nice catch! Thanks!

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 3]
```diff
@@ -30,191 +30,190 @@ Given that etcd often handles sensitive data, improving and maintaining security
...
 has been compacted, see [kube-apiserver v1.21
 code](https://github.com/kubernetes/kubernetes/blob/v1.21.0/staging/src/k8s.io/client-go/tools/cache/reflector.go#L302-L312)),
-often causing slowness in reads from overloaded etcd servers (see [github
-issue](https://github.com/etcd-io/etcd/issues/12678)). In such case,
-kube-apiserver tracing warns as below:
+often causing slowness in reads from overloaded etcd servers (see [GitHub
+issue](https://github.com/etcd-io/etcd/issues/12678)). In such cases,
+`kube-apiserver` tracing warns as below:
 
 > "List" url:/api/v1/pods,user-agent... (started: ...) (total time: 1.208s): Trace[...]: [1.208s] [1.204s] Writing http response done count:4346
```
Comment (nate-double-u) [inline content/en/blog/2021/announcing-etcd-3.5.md:124]:
```suggestion
`"List" url:/api/v1/pods,user-agent... (started: ...) (total time: 1.208s): Trace[...]: [1.208s] [1.204s] Writing http response done count:4346`
```

or a ``` fence

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 4]
```diff
@@ -30,191 +30,190 @@ Given that etcd often handles sensitive data, improving and maintaining security
...
 
-And etcd warns
+And etcd warns as follows:
 
 > etcdserver: read-only range request key:"/registry/pods/" range_end:"/registry/pods0" revision:... range_response_count:500 size:291984 took too long (723.099118ms) to execute
```
Comment (nate-double-u) [inline content/en/blog/2021/announcing-etcd-3.5.md:128]:
```suggestion
`etcdserver: read-only range request key:"/registry/pods/" range_end:"/registry/pods0" revision:... range_response_count:500 size:291984 took too long (723.099118ms) to execute`
```

or a ``` fence

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 5]
```diff
@@ -30,191 +30,190 @@ Given that etcd often handles sensitive data, improving and maintaining security
...
 
 **etcd now supports built-in log rotation** that configures rotate thresholds,
-compression algorithms, etc. -- see [hexfusion@'s code change (from Red
-Hat)](https://github.com/etcd-io/etcd/pull/12774).
+compression algorithms, etc. For more information, see [hexfusion@ of Red Hat's code change](https://github.com/etcd-io/etcd/pull/12774).
 
-**etcd now emits more detailed tracing information for expensive requests**, as such:
+**etcd now emits more detailed tracing information for expensive requests**, such as:
 
 > "caller":"traceutil/trace.go:116","msg":"trace[123] range","detail":"{range_begin:foo; range_end:fooo; response_count:100000; response_revision:191496;}","duration":"132.449773ms","start":"...:32.611-0700","end":"...:32.744-0700","steps":["trace[123] step 'range keys from bolt db' (duration: 92.521911ms)","trace[123] step 'filter and sort the key-value pairs' (duration: 22.789099ms)"]}
```
[the remark this answers is not in the document]
Comment (gyuho) [reply content/en/blog/2021/announcing-etcd-3.5.md:55]: @nate-double-u Feel free to edit! Yes, these are structured logging.

**Review by nate-double-u — APPROVED** [review]
LGTM, thanks @kapunahelewong!

**Pull request merged**